addons/mrp/views/mrp_workcenter_views.xml

```diff
@@ -388,6 +387,15 @@
                                 <separator string="Description"/>
                                 <field name="note" nolabel="1" placeholder="Description of the work center..."/>
                             </page>
+                            <page string="Specific Capacities" name="capacity">
+                                <field name="capacity_ids" context="{'default_workcenter_id': id, 'default_company_id': company_id}">
+                                    <tree editable="bottom">
+                                        <field name="product_id" required="1"/>
+                                        <field name="product_uom_id" readonly="1"/>
```

Comment:
You already have `required=True` on the model field and related fields are already readonly by default